go-backend/service/user_http_test.go

```diff
@@ -67,9 +69,9 @@ func (suite *UsersHandlerTestSuite) TestListUsersWhenDBFailure() {
 	suite.dbMock.AssertExpectations(suite.T())
 }
 
-func makeHTTPCall(method, path, body string, handlerFunc http.HandlerFunc) (recorder *httptest.ResponseRecorder) {
+func makeHTTPCall(method, path, requestUrl, body string, handlerFunc http.HandlerFunc) (recorder *httptest.ResponseRecorder) {
```

Comment:
how is path and requestURL different ?

Reply:
i got that, but whatever you are trying to request for eg: /users/1234

you can request on it as well as define the router on that too. That will work, right ?

Reply:
Yes

Reply:
closing this PR as not needed.

Reply:
@iamanujvrma I am facing this issue.  Are you going to close this PR or merge it ?